.jenkins/pytorch/common_utils.sh

```diff
@@ -117,6 +117,7 @@ function clone_pytorch_xla() {
     pushd xla
     # pin the xla hash so that we don't get broken by changes to xla
     git checkout "$(cat ../.github/ci_commit_pins/xla.txt)"
+    git submodule update --init --recursive
```

Comment:
I usually do `git submodule sync; git submodule update --init --recursive`, any reason we don't need to do `sync` ?

Reply:
The sync should only be necessary if the submodule hashes differ between the default branch (master) and the checkout you want.

Reply:
that's right. I added it to be safe.